libs/spark/src/Unstable_Checkbox/Unstable_CheckboxIcon.tsx

```diff
@@ -0,0 +1,149 @@
+import * as React from 'react';
+import clsx from 'clsx';
+import makeStyles from '../makeStyles';
+import unstable_createSvgIcon from '../unstable_createSvgIcon';
+
+const useStyles = makeStyles<
+  'root' | 'checked' | 'indeterminate' | 'box' | 'check' | 'dash'
+>(
+  (theme) => ({
+    /* Styles applied to the root element. */
+    root: {
+      display: 'flex',
+      position: 'relative' as const,
+      borderRadius: 2,
+      color: theme.unstable_palette.neutral[200],
+      // Adjust for irregular svg size
```

Comment:
The svg for the box is 17px x 17px, which is thus it's viewbox (0 0 17 17). The typical size assigned is 24px x 24px (for the standard 0 0 24 24 viewbox). So without these, it would scale up to that size.